### PR TITLE
fix: [CI-4346]: add replace config for auth

### PR DESCRIPTION
**Pull request by aman-harness**
You can use the following comments to re-trigger PR Checks

- Compile: `trigger compile`
- runAeriformCheck: `trigger AeriformCheck`
- CodeFormat: `trigger codeformat`
- MessageMetadata: `trigger messagecheck`
- Recency: `trigger recency`
- BuildNumberMetadata: `trigger buildnum`
- runDockerizationCheck: `trigger dockerizationcheck`
- runAuthorCheck: `trigger authorcheck`
- Checkstyle: `trigger checkstyle`
- PMD: `trigger pmd`
- TI-bootstrap: `trigger ti0`
- TI-bootstrap1: `trigger ti1`
- TI-bootstrap2: `trigger ti2`
- TI-bootstrap3: `trigger ti3`
- TI-bootstrap4: `trigger ti4`
- FunctionalTest1: `trigger ft1`
- FunctionalTest2: `trigger ft2`
- CodeBaseHash: `trigger codebasehash`

---
This change is [Reviewable](https://reviewable.io/reviews/harness/harness-core/32710)

### CODE REVIEW

File: build.properties [hunk 1]
```diff
@@ -1,6 +1,6 @@
 build.majorVersion=1
 build.minorVersion=0
 build.number=75001
-build.patch=000
+build.patch=001
```
Comment (harshjain12) [inline build.properties:4]: @nataraja2018 Is it fine ?
I think we don't allow patch in QA, Can you please confirm it ?

Comment (aman-harness) [reply build.properties]: I have updated the build number

**Comment by aman-harness** [comment]
trigger messagecheck

**Comment by aman-harness** [comment]
trigger messagecheck

**Pull request merged**